lib/main-app-helper.js

```diff
@@ -0,0 +1,173 @@
+
+var nodeHelpers = require('./node-helpers');
+var fs = require('fs-extra');
+var http = require('http');
+var https = require('https');
+var httpProxy = require('http-proxy');
+var async = require('async');
+
+module.exports = function(cozyLight){
+
+  var LOGGER = cozyLight.logger;
```

Comment:
i did it. Something eventually more interesting to do there, is to switch from printit to npmlogger. There is more convienients  method in it.
Such as i wanted to -v -vv CLI options, but printit does not provide such log level filters, i believe.